quiz.py

```diff
@@ -0,0 +1,144 @@
+import random
+from abc import ABCMeta, abstractmethod
+from sqlalchemy.sql.expression import exists
+from sqlalchemy.sql.functions import func
+from models.painter import Painter
+from models.painting import Painting
+from models.saint import Saint
+
+
+class MultipleChoiceQuestion:
+    def __init__(self, question, correct_answer, option_list):
+        self.question_dict = question
+        self.question = question.values()[0]
+        self.correct_answer = correct_answer
+        self.option_list = option_list
```

Comment:
other_answers is not accurate either, cause it implies that the correct answer is not included.